src/services/CertificateGenerationService.ts

```diff
@@ -100,9 +101,18 @@ class CertificateGenerationService {
       vehicleTestRes = "rwt";
     } else if (this.isTestTypeAdr(testResult.testTypes)) {
       vehicleTestRes = "adr_pass";
+    } else if (testResult.vehicleType === "hgv" && testType.testResult === "pass" && isTestStationWelsh) {
+      vehicleTestRes = testResult.vehicleType + "_" + testType.testResult + "_bilingual";
+      console.log("** THIS IS THE vehicleTestRes in the else if " + vehicleTestRes);
     } else {
       vehicleTestRes = testResult.vehicleType + "_" + testType.testResult;
     }
+
```

Comment:
Are all of these logs still needed? Or were they only needed while debuging, if so you could change them to console.debug.